### PR TITLE
🌐 style: Add Vietnamese files and add the vi-VN option in the General Settings

### PR DESCRIPTION
**Pull request by PearFarmer**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [x] 🔨 chore: Add Vietnamese files and add the vi-VN option in the General Settings
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change
- Inside `locales` folder, I added a folder `vi-VN`. Inside that folder, I added the same json files used to support languges: 
   - chat.json
   - common.json
   - empty.json
   - error.json
   - market.json
   - migration.json
   - plugin.json
   - setting.json
   - tool.json
   - welcom.json
 - Inside each of those json file, I translated all the pairs of  value to support Vietnamese.
 - Inside file `src/locales/resources.ts`, I added code to support Vietnames, loading data from folder vi-VN.

#### 📝 Demo: 
demo-better

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { SpeedInsights } from '@vercel/speed-insights/next';
```
Comment (arvinxx) [inline src/app/layout.tsx]: please don't introduce it

File: package.json
Comment (arvinxx) [inline package.json:1]: please revert this change

**Review by arvinxx — CHANGES_REQUESTED** [review]
please add the `vi-VN` into the `i18nrc.js` and run `npm run i18n`.

https://github.com/lobehub/lobe-chat/blob/main/.i18nrc.js#L7

we won't update the i18n manually. So we use `lobe-i18n` to auto translate locale.

You can check  simmiliar pr like: https://github.com/lobehub/lobe-chat/pull/759

File: src/app/layout.tsx (hunk 1 above)
Comment (PearFarmer) [reply src/app/layout.tsx]: ok, let me remove it

**Comment by CongLeSolutionX** [comment]
> please add the `vi-VN` into the `i18nrc.js` and run `npm run i18n`.
> 
> https://github.com/lobehub/lobe-chat/blob/main/.i18nrc.js#L7
> 
> we won't update the i18n manually. So we use `lobe-i18n` to auto translate locale.
> 
> You can check simmiliar pr like: #759

Thanks for your feedback, let me update this PR

File: package.json
Comment (PearFarmer) [reply package.json:1]: ok

**Comment by CongLeSolutionX** [comment]
> please add the `vi-VN` into the `i18nrc.js` and run `npm run i18n`.
> 
> https://github.com/lobehub/lobe-chat/blob/main/.i18nrc.js#L7
> 
> we won't update the i18n manually. So we use `lobe-i18n` to auto translate locale.
> 
> You can check simmiliar pr like: #759

Hi @arvinxx,
I follow your instructions to use `.i18nrc.js` to translate all files in the vi-VN folder.
`.i18nrc.js` also picks up the missing string "downloading" as well, so I add those changes to this PR.

**Review by arvinxx — APPROVED** [review]
Thanks for your great job!

**Pull request merged**